mmv1/templates/tgc/resource_converters.go.erb

```diff
@@ -115,6 +114,7 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_cloudfunctions_function":                  {resourceConverterCloudFunctionsCloudFunction()},
 		"google_monitoring_notification_channel":          {monitoring.ResourceConverterMonitoringNotificationChannel()},
 		"google_monitoring_alert_policy":                  {monitoring.ResourceConverterMonitoringAlertPolicy()},
+		"google_vertex_ai_dataset":                        {vertexai.ResourceConverterVertexAIDataset()},		
```

Comment:
Drive-by comment - please ensure all the whitespace changes are removed before merging.

Reply:
this will help avoid merge conflicts as well.